src/components/tabs/tabs.stories.mdx

```diff
@@ -86,8 +86,6 @@ The tabs widget also allows you to select a tab on page load. By default this is
 
 ### With Link as a Tab
 
-To set a different tab on page load pass a `tabId` to the `selectedTabId` prop as shown in the example below.
```

Comment:
**question(non-blocking):** was this meant to be removed in this PR?

Reply:
Yes, this description was a repeat of the story below it so it seemed to have been copied over by mistake.